Sources/BraintreeCore/BTAppContextSwitcher.swift

```diff
@@ -13,7 +13,10 @@ import UIKit
     /// The URL scheme to return to this app after switching to another app or opening a SFSafariViewController.
     /// This URL scheme must be registered as a URL Type in the app's info.plist, and it must start with the app's bundle ID.
     public var returnURLScheme: String = ""
-    
+
+    /// The URL to use for the PayPal universal link flow. Must contain the path `braintree-payments`.
```

Comment:
Do we want to require this specific path or another one for this feature?

Reply:
I was thinking similar to[ URL schemes](https://developer.paypal.com/braintree/docs/guides/venmo/client-side/ios/v6#register-a-url-type), we want them to include some path specific to BT payments, but it could be whatever they want

Reply:
I don't think we should validate that it contains `braintree-payments`. They might have an existing path they want to use for their checkout flow

Reply:
Sounds great - I can update this to the following if that sounds good:
```suggestion
    /// The URL to use for the PayPal universal link flow. Must be dedicated to Braintree app switch returns.
```

Reply:
Love it!